Jenkinsfile_pipeline_test

```diff
@@ -13,7 +13,7 @@
 
 properties([
   parameters([
-    string(name: 'LIB_VERSION', defaultValue: 'master', description: 'Branch name of pipeline library to use')
+    string(name: 'LIB_VERSION', defaultValue: 'test-with-01-cluster', description: 'Branch name of pipeline library to use')
```

Comment:
won't help what u are trying to do. u need something like https://github.com/hmcts/cnp-plum-recipes-service/commit/e4a29deb170c4deed118d59c0f18985ed1dd09e6#diff-a31d2a7e600e89b405cd675bd2f5bc6371e64b20cab9d32ab2a0279209a1882fL3